esp/esp/program/models/class_.py

```diff
@@ -588,6 +588,18 @@ def already_passed(self):
             if time_passed > timedelta(0):
                 return True
         return False
+
+    def start_time_prefetchable(self):
+        """Like self.start_time(), but can be prefetched.
```

Comment:
The comment here is misleading--it makes me think that you can drop this in anywhere start_time() is used, but they return different kinds of things. (Also, are there other places in the code that might want start_time_prefetchable() instead?) Maybe the name of one or the other wants to be different too.

Question: why use meeting_times.all() here instead of getting it from the cache with get_meeting_times()?